content/en/blog/_posts/2018-07-12-resizing-persistent-volumes-using-kubernetes.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: blog
+title: 'Resizing Persistent Volumes using Kubernetes'
+date: 2018-07-12
+---
+
+**Author**: Hemant Kumar (Red Hat)
+
+**Editor’s note: this post is part of a [series of in-depth articles](https://kubernetes.io/blog/2018/06/27/kubernetes-1.11-release-announcement/) on what’s new in Kubernetes 1.11**
+
+In Kubernetes v1.11 the persistent volume expansion feature is being promoted to beta. This feature allows users to easily resize an existing volume by editing the PersistentVolumeClaim object. Users no longer have to manually interact with the storage backend or delete and recreate PV and PVC objects to increase the size of a volume. Shrinking persistent volumes is not supported.
+
+Volume expansion was introduced in v1.8 as an Alpha feature, and versions prior to v1.11 required enabling the feature gate, `ExpandPersistentVolumes`, as well as the admission controller, `PersistentVolumeClaimResize` (which prevents expansion of PVCs whose underlying storage provider does not support resizing). In Kubernetes v1.11+, both the feature gate and admission controller are enabled by default.
+
+Although the feature is enabled by default, a cluster admin must opt-in to allow users to resize their volumes. Kubernetes v1.11 ships with volume expansion support for the following in-tree volume plugins: AWS-EBS, GCE-PD, Azure Disk, Azure File, Glusterfs, Cinder, Portworx, and Ceph RBD. Once the admin has determined that volume expansion is supported for the underlying provider, they can make the feature available to users by setting the `allowVolumeExpansion` field to `true` in their StorageClass object(s). Only PVCs created from that StorageClass will be allowed to trigger volume expansion.
```

Comment:
`StorageClass` is also an API object, possibly in " `" quotes like the rest of the API objects.
https://kubernetes.io/docs/concepts/storage/storage-classes/